### PR TITLE
Add monitoring capabilities to Kafka straight out of the box

### PR DESCRIPTION
**Pull request by rik-v**
This change pulls in the JMX exporter directly to deploying zookeeper.

### CODE REVIEW

File: containter_host.tf [hunk 1]
```diff
@@ -70,6 +76,7 @@ resource "null_resource" "cluster" {
     # Bootstrap script called with private_ip of each node in the cluster
     inline = [
       "chmod +x /home/${var.user}/bootstrap-cluster.sh",
+      "chmod 777 /home/${var.user}/jmxconfig.yml",
```
Comment (loafoe) [inline containter_host.tf]: Why 777?

**Review by loafoe — CHANGES_REQUESTED** [review]

File: containter_host.tf (hunk 1 above)
Comment (johan-horst) [reply containter_host.tf]: Changed to 755, no reason ;-)

File: main.tf [hunk 2]
```diff
@@ -47,6 +47,12 @@ resource "hsdp_container_host_exec" "cluster" {
     source      = "${path.module}/scripts/bootstrap-cluster.sh"
     destination = "/home/${var.user}/bootstrap-cluster.sh"
   }
+  
+  file {
```
Comment (loafoe) [inline main.tf:51]: This should be dynamic no?

File: main.tf [hunk 3]
```diff
@@ -47,6 +47,12 @@ resource "hsdp_container_host_exec" "cluster" {
...
+    source      = "${path.module}/scripts/jmxconfig.yml.tmpl"
```
Comment (loafoe) [inline main.tf:52]: And is it a template with requires variable subtitutions? or just a static .yml?

Comment (johan-horst) [reply main.tf:52]: There is one variable that is substituted.

Comment (loafoe) [reply main.tf:52]: BTW, this module resolves variables in a script on the server. This was before I learned about `templatefile` which can render the template on the Terraform and basically just push parameterless scripts to the server.  Some day we should rewrite this :)

File: main.tf (hunk 2 above)
Comment (johan-horst) [reply main.tf:51]: as discussed, it is nice, but not necessary

**Pull request merged**